recipes/default.rb

```diff
@@ -22,6 +22,10 @@
     template "/etc/pam.d/su" do
       cookbook ulimit['pam_su_template_cookbook']
     end
+    cookbook_file "/etc/pam.d/sudo" do
+      source "sudo"
```

Comment:
Please make `source` and `cookbook` optional/attribute driven here, so that if someone wants to do this differently, they can.